src/core/editor.coffee

```diff
@@ -167,12 +173,37 @@ class Editor
       )
     )
 
+  _splitDelta: (delta, point) ->
+    return [
+      delta.slice(0, point),
+      delta.slice(point),
+    ]
+
+  _mergeDelta: (deltaA, deltaB) ->
+    return new Delta(deltaA.ops.concat(deltaB.ops))
```

Comment:
A helper is not necessary here since it's only called once and is so short. It'd be better to have this inline in the caller with a short comment ex.

```coffeescript
diff = new Delta(diffLeft.ops.concat(diffRight.ops))   # merge diffLeft and diffRight
```